package.json

```diff
@@ -56,21 +56,21 @@
     "@mongodb-js/oidc-mock-provider": "^0.6.2",
     "@mongodb-js/prettier-config-compass": "^0.5.0",
     "@mongodb-js/tsconfig-compass": "^1.0.1",
+    "@mongodb-js/eslint-config-devtools": "^0.9.4",
     "@types/chai": "^4.2.21",
     "@types/express": "^4.17.17",
     "@types/mocha": "^9.0.0",
     "@types/node": "^18.13.0",
     "@types/oidc-provider": "^8.1.1",
     "@types/sinon-chai": "^3.2.5",
-    "@typescript-eslint/eslint-plugin": "^5.52.0",
-    "@typescript-eslint/parser": "^5.54.1",
+    "@typescript-eslint/eslint-plugin": "^4.28.4",
```

Comment:
Not sure why we still have to have these in the repo itself. I think these can just be deps of @mongodb-js/eslint-config-devtools? I'll PR that separately.

Reply:
Same for "@typescript-eslint/parser" and "eslint-plugin-mocha" and maybe even "eslint".

Reply:
https://github.com/mongodb-js/oidc-plugin/pull/125